app/src/main/java/com/triviagenai/triviagen/ui/theme/Type.kt

```diff
@@ -35,6 +35,14 @@ val Typography = Typography(
         letterSpacing = 0.sp,
         color = Color.White
     ),
+    titleMedium = TextStyle(
+        fontFamily = karmaFontFamily,
+        fontWeight = FontWeight.W600,
+        fontSize = 24.sp,
```

Comment:
Would it be better to have `sp` in the `dimens` resource file?

Reply:
No, this is the file containing text styles, and it is the only place in whole code where sp is used, so I think it is unnecesary.